environs/broker.go

```diff
@@ -105,6 +105,11 @@ type StartInstanceParams struct {
 // StartInstanceResult holds the result of an
 // InstanceBroker.StartInstance method call.
 type StartInstanceResult struct {
+	// DisplayName is an optional human-readable string that's used
+	// for display purposes only. When empty, the output defaults to
+	// StartInstanceResult.Instance.Id()
+	DisplayName string
```

Comment:
This doesn't appear to be used - it needs to be fed into the SetInstanceInfo() API call.
(with the appropriate api versioning)

Reply:
Now available within the provider code